### ISSUE TITLE
可否让玩家在车辆中坐下

### ISSUE THREAD

**Issue by DeadBoy2537** [jonafanho/Minecraft-Transit-Railway#369]
你们可以在这个模组里添加一个能够就座的座位吗？就是说，玩家可以自由选择座位，或者站立在车厢中。

**Comment by JasperkangYT** [comment]
我覺得真的需要這個......

**Comment by XHG78999** [comment]
目前座椅是一整个实体，要是坐只能做一个人。

**Comment by DeadBoy2537** [comment]
> 目前座椅是一整个实体，要是坐只能做一个人。

就那个mtr:seat吗

**Comment by XHG78999** [comment]
> > 现在是一个整体实体，只想坐一个人。
> 
> 就在那个mtr:seat吗

是的。我查了一下源代码，一条椅子只有一个实体EntitySeat。当然我们可不希望像隔壁RealBug，啊不，RealTrainMod一样每一个部件都有一个实体。

**Comment by DeadBoy2537** [comment]
> > > 现在是一个整体实体，只想坐一个人。
> > 
> > 
> > 就在那个mtr:seat吗
> 
> 是的。我查了一下源代码，一条椅子只有一个实体EntitySeat。当然我们可不希望像隔壁RealBug，啊不，RealTrainMod一样每一个部件都有一个实体。

那确实，rtm太卡了

**Comment by XHG78999** [comment]
> > > > 现在是一个整体实体，只想坐一个人。
> > > 
> > > 
> > > 就在那个mtr:seat吗
> > 
> > 是的。我查了一下源代码，一条椅子只有一个实体EntitySeat。当然我们可不希望像隔壁RealBug，啊不，RealTrainMod一样每一个部件都有一个实体。
> 
> 那确实，rtm太卡了

主要的问题是由于我们不知道贴图有多少座位，不好匹配。
再说现在我们这么多车，位置不是都一样的。。。。

**Comment by Leoparking** [comment]
现在这个设定已经够好了吧？可以在车厢里移动，走到某个座位的位置站着就相当于入座了吧

**Comment by hahahotsoup** [comment]
有没有办法能让附加包作者在模型里用特定名称的object声明这是一个座位？
